base.js

```diff
@@ -49,10 +50,16 @@ class Base extends EventEmitter {
   }
 
   loadConf (c, group = null) {
-    _.merge(
-      this.conf,
-      this.getConf(this.ctx.env, group, `${this.ctx.root}/config/${c}.json`)
-    )
+    const fprefix = this.ctx.env === 'test' ? 'test' : ''
+    const dirname = join(this.ctx.root, 'config')
+
+    let confpath = join(dirname, `${c}.json`)
+    const testpath = join(dirname, `${fprefix}.${c}.json`)
+    if (fprefix && fs.existsSync(testpath)) {
```

Comment:
hmmm... this would lead to a silent error when a testfile is not there, but the user starts the service with the --env=test arg

Reply:
on the other side, this is intented for backwards compat and where a seperate test* conf file is not requried

Reply:
yep the idea was to keep backward compatibility also so nothing breaks if test file doesn't exist